File: LeetCode/Easy/1822-sign-of-the-product-of-an-array/1822-sign-of-the-product-of-an-array.java
class Solution {
    public int arraySign(int[] nums) {
        double ret = 1;
        for (int i = 0; i < nums.length; ++i){
            ret *= nums[i];
        }
        return ret > 0 ? 1 : ret < 0 ? -1 : 0;
    }
}

File: LeetCode/Easy/1822-sign-of-the-product-of-an-array/1822-sign-of-the-product-of-an-array.go
func arraySign(nums []int) int {
    var ret float64 = 1
    for i := 0; i < len(nums); i++ {
        ret *= float64(nums[i])
    }
    if ret > 0 {return 1}
    if ret < 0 {return -1}
    return 0
}